config/game_options.txt

```diff
@@ -305,9 +305,21 @@ DEFAULT_REP_VALUE 5
 ## Uncomment to allow players to see the set odds of different rounds in secret/random in the get server revision screen. This will NOT tell the current roundtype.
 #SHOW_GAME_TYPE_ODDS
 
+
+## Dynamic options
 ## Uncomment to enable dynamic ruleset config file.
 DYNAMIC_CONFIG_ENABLED
 
+## The minimum station integrity, as a percentage value, for potentially destructive crew antags to roll.
+## Basically, if the station is blown to bits, it won't spawn traitors to come in and pick apart what's left.
+## Set to 0 to disable station integrity checks.
+DYNAMIC_MINIMUM_STATION_INTEGRITY 95
```

Comment:
95 is super high for a default. The SM could go boom boom and now nothing can roll. Anything lower than 85-90 would be much better.

Reply:
I actually got this value by testing locally - I barely reached 95 after dropping a maxcap on medbay and the bridge.

Reply:
Changed to 85